proxy.js

```diff
@@ -0,0 +1,42 @@
+//ex. proxy server which I'd use in place of my index.js file if I were not using the cors npm package
+
+const express = require('express'),
+    request = require('request'),
+    bodyParser = require('body-parser'),
+    app = express();
+
+const myLimit = typeof(process.argv[2]) != 'undefined' ? process.argv[2] : '100kb';
+console.log('Using limit: ', myLimit);
+
+app.use(bodyParser.json({limit: myLimit}));
+
+app.use(express.static('public'));
+
+app.all('*', function (req, res, next) {
+
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Methods", "GET, PUT, PATCH, POST, DELETE");
+    res.header("Access-Control-Allow-Headers", req.header('access-control-request-headers'));
+
+    if (req.method === 'OPTIONS') {
```

Comment:
I'm not 100% what this portion is checking..  Is it forwading the request or checking that there isn't an error?

Reply:
it's manually doing what the cors npm package does!